src/components/funFacts/funfacts.ts

```diff
@@ -0,0 +1,26 @@
+export interface FunFact {
+    id: number;
+    prompt: string;
+    detail: string;
+}
+
+export const funFacts: FunFact[] = [
+    {
+        id: 1,
+        prompt: "Mars is named after the Roman god of war.",
+        detail: "The month march was also named after him. Mars's moons, called Phobos and Deimos, are named after the gods of fear",
+    },
+
+    {
+        id: 2,
+        prompt: "A year on Mars is not the same as on Earth?",
```

Comment:
Grammar (not a question)
```suggestion
        prompt: "A year on Mars is not the same as on Earth!",
```